openfisca_core/columns.py

```diff
@@ -19,6 +19,11 @@ def N_(message):
 
 year_or_month_or_day_re = re.compile(ur'(18|19|20)\d{2}(-(0?[1-9]|1[0-2])(-([0-2]?\d|3[0-1]))?)?$')
 
+# Possible values for period_behavior
+MONTH = u'month'
+YEAR = u'year'
+PERMANENT = u'permanent'
```

Comment:
```py
from openfisca_core.enumerations import Enum
ENUM = Enum(['month', 'year'])
raise ValueError('Computation requested with wrong period unit for variable {} ({} instead of {})'.format('rsa', 'month', ENUM['year']))

>>> ValueError: Computation requested with wrong period unit for variable rsa (month instead of 1)
```

This is for me a huge drawback to the use of the current polyfill of `Enum`. Also, writing `ENUM['year']` is super heavy.

If we can use a better polyfill, ok, but the current one is not good enough.